### ISSUE TITLE
rustdesk 1.3.4 on opensuse leap 15.6 doesnt show gui main window any more, only kde tray icon and kde symbol, 1.3.3 worked flawlessly

### ISSUE THREAD

**Issue by aabittner** [rustdesk/rustdesk#10180]
### Bug Description

rustdesk 1.3.4 on opensuse leap 15.6 doesnt show gui main window any more, only kde tray icon and kde symbol, 1.3.3 worked flawlessly

just upgraded via the .rpm provided here in github to 1.3.4 x64 opensuse rpm
the service is running fine and enabled, but the main gui only shows a KDE app symbol that has rightclick context menu "recent connections" and "forget recent connections"... and the normal KDE stuff resize move around etc but I can not manage to make it show the normal GUI of rustdesk with my ID with the remote IDs and thumbnails etc.

reverting back to 1.3.3 suse rpm brings back functionality normally.

### How to Reproduce

use opensuse leap 15.6 x64 KDE destkop, and use rustdesk 1.3.3 first then move to 1.3.4

### Expected Behavior

1.3.4 should just work normally displaying its main GUI just like 1.3.3 did before

### Operating system(s) on local (controlling) side and remote (controlled) side

opensuse 15.6 x64 with KDE plasma desktop

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.4, local gui issue already

### Screenshots

-

### Additional Context

nothing fancy to report. simple system simple stuff

**Comment by mujeebcpy** [comment]
same here. i am using arch linux. 
i tested aur package and appimage from github. everything fails to show gui main window

here is the debug log from terminal

```
Cannot load libcuda.so.1
Cannot load libcuda.so.1

** (rustdesk:27678): CRITICAL **: 00:32:04.923: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

** (rustdesk:27678): CRITICAL **: 00:32:04.924: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

** (rustdesk:27678): CRITICAL **: 00:32:04.925: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

** (rustdesk:27678): CRITICAL **: 00:32:04.926: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

** (rustdesk:27678): CRITICAL **: 00:32:04.926: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found
Linux desktop, XDG_CURRENT_DESKTOP: KDE
Try setting transparent
flutter: launch args: []
flutter: initializing FFI main
flutter: _appType:main,info1-id:f8026e1185824208b7da85b0ae7c39c1,info2-name:EndeavourOS,dir:/home/mujeeb/Documents
flutter: _globalFFI init
flutter: registerEventHandler callback_query_onlines address book peer
flutter: registerEventHandler load_address_book_peers address book peer
flutter: registerEventHandler callback_query_onlines group peer
flutter: registerEventHandler load_group_peers group peer
flutter: registerEventHandler callback_query_onlines recent peer
flutter: registerEventHandler load_recent_peers recent peer
flutter: registerEventHandler callback_query_onlines fav peer
flutter: registerEventHandler load_fav_peers fav peer
flutter: registerEventHandler callback_query_onlines discovered peer
flutter: registerEventHandler load_lan_peers discovered peer
flutter: _globalFFI init end
flutter: registerEventHandler native_ui native_ui
flutter: registerEventHandler check_software_update_finish check_software_update_finish
flutter: Linux distro is not linuxmint, and desktop is not mate, so we build virtual window frame.
flutter: restore lpos: 800.0/600.0, offset:345.0/87.0, isMaximized: false, isFullscreen: false
flutter: handled by uni links: false
flutter: [MultiWindowHandler] active window changed: {0}
```

**Comment by Ziris85** [comment]
Echoing this as well, coming from Manjaro using the appimage. 1.3.2 version works fine, I jumped to 1.3.4 and same behavior as OP: icon in taskbar but no window.

**Comment by davidebeatrici** [comment]
Same on OpenMandriva.

**Comment by sahsanu** [comment]
Same here, using KDE Neon 6.2 on X11 and installing from provided deb package.

**Comment by rizalmart** [comment]
Same here using Puppy Linux and Slackware, it doesn't show the rustdesk main window. Please fix it.

**Comment by POLFODEX** [comment]
I confirm that on my Linux Debian 12 with the KDE shell, the main window does not appear. The application also does not show up in the tray. I suspect that in an attempt to remove the black corners at the rounded corners of the window, they might have used Wayland instead of X11.

**Comment by C-O-D** [comment]
Same here on Manjaro installed from AUR...

**Comment by rustdesk** [comment]
Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
 @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner

**Comment by C-O-D** [comment]
> Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> 
> The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> 
> @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner

Flatpak does work herw on Manjaro which it didn't before.
The 
"Rounded corners are opaque black after 1.3.3 (#10121)"
still not working...

**Comment by rustdesk** [comment]
> Rounded corners are opaque black after 1.3.3

This is not our high priority comapred with no window shown.

**Comment by rustdesk** [comment]
> > Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> > The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> > @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner
> 
> Flatpak does work here on Manjaro which it didn't before. The "Rounded corners are opaque black after 1.3.3 (#10121)" still not working...

Flatpak is not ready, please test AppImage.

**Comment by mujeebcpy** [comment]
> Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> 
> The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> 
> @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner

tested the nightly appimage. it works fine in endevour linux kde. thanks

**Comment by C-O-D** [comment]
> > > Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> > > The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> > > @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner
> > 
> > 
> > Flatpak does work here on Manjaro which it didn't before. The "Rounded corners are opaque black after 1.3.3 (#10121)" still not working...
> 
> Flatpak is not ready, please test AppImage.

Yo, that's nice and working 👍

**Comment by POLFODEX** [comment]
> Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> 
> The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> 
> @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner

I tested the nightly .deb version, everything works perfectly.
P.S. The window corners are rounded and transparent.
![edge](https://github.com/user-attachments/assets/4b3bd02b-028f-46b9-8727-e6d588e1e575)
![edge2](https://github.com/user-attachments/assets/410f5317-8d8d-4655-bdc3-e537c1f867f2)

**Comment by aabittner** [comment]
> Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> 
> The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> 
> @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner

opensuse rpm ( abba717760e00235603393abc90f21cdf82f9c92e4d9e9018a61d94103969acd ) works now again on a opensuse leap 15.6 x64 machine, showing its main GUI fine again and able to connect to remote rustdesk IDs. thanks for the quick fix.

**Comment by sahsanu** [comment]
> Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix

Tested deb package on KDE Neon 6.2 using X11 and the fix is working pretty fine.

Thank you very much.

**Comment by jvschiavo** [comment]
The 1.3.5 AppImage also solved this problem for me (on 1.3.4 AppImage) on Debian.

**Comment by rizalmart** [comment]
> > Could you test https://github.com/rustdesk/rustdesk/releases/nightly? We put a fix
> > The deb/rpm/pkg and appimage are ready, flatpak are not ready yet.
> > @C-O-D @POLFODEX @rizalmart @sahsanu @davidebeatrici @Ziris85 @mujeebcpy @aabittner
> 
> opensuse rpm ( abba717760e00235603393abc90f21cdf82f9c92e4d9e9018a61d94103969acd ) works now again on a opensuse leap 15.6 x64 KDE machine, showing its main GUI fine again and able to connect to remote rustdesk IDs. thanks for the quick fix.

Finally it now works on Puppy Linux and Slackware. Thanks
